lib/hooks/docker-hub-automated-build/templates/default.html.haml

```diff
@@ -0,0 +1,4 @@
+%div
+  #{payload.push_data.pusher} pushed #{payload.repository.repo_name}:#{payload.push_data.tag}
+  %br
+  #{payload.repository.repo_url}
```

Comment:
How about using this URL for the link of repo name (`#{payload.repository.repo_name}`) above?  In that way, we can remove this 2nd line.